### DISCUSSION TITLE
No option to disable `declarationMap`

### DISCUSSION THREAD

**Discussion by tjzel** [callstack/react-native-builder-bob#631] [Ideas]
### Description

There's one to disable source maps but not declaration maps.

### Packages

- [ ] create-react-native-library
- [X] react-native-builder-bob

### Selected options

irrelevant

### Link to repro

irrelevant

### Environment

irrelevant

**Comment by satya164** [comment]
We should disable that if sourcemaps are disabled. PR welcome.

**Comment by atlj** [comment]
Hey I think this issue is more suitable as a discussion so transferring this.

**Comment by tjzel** [comment]
I was sceptical at first to embed declaration maps so suddenly, but it works like a charm so I think this option isn't really needed atm.
